tests/test_user.py

```diff
@@ -0,0 +1,50 @@
+from app import app
+import unittest
+import json
+
+class TestUserLogin(unittest.TestCase):
+	"""Test class for user login"""
+	def setUp(self):
+		"""Creates the app as a test client"""
+		app.testing = True
+		self.app = app.test_client()
+		self.new_user_info = {
```

Comment:
Remove this from this method, You will not need it everytime you call the setup function.
You can have a login and sign in functions  that are called when needed.

Reply:
Done.